internal/cache/controller_obj_cache.go

```diff
@@ -133,11 +133,38 @@ func (c *AviObjCache) AviObjCachePopulate(client *clients.AviClient, version str
 	return vsCacheCopy, allVsKeys, nil
 }
 
+//TODO: Deperecate this function in future release.
+//This function list EVH child VS to be deleted which contain namespace in its un-encoded name.
+func (c *AviObjCache) listEVHChildrenToDelete(vs_cache_obj *AviVsCache, childUuids []string) ([]NamespaceName, []string) {
```

Comment:
@sudswasavi @monotosh-avi @chauhanshubham : Can you please check this function from GC point of view? will this objects or objects declared in parent function will occupy mem even after function call exited?

Reply:
i think these variables are local to the function .. so should get cleaned up. 
Though @chauhanshubham had observed that there are some memory optimisations that has to be done. So we need to take a look at that in general.